# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1466528832 14400
#      Tue Jun 21 13:07:12 2016 -0400
# Node ID 63f8e75e30431de39eb93d32d0cbf88c6213c6ff
# Parent  627f582edbe38744d12adb2d4cb9e61fe271c504
shared-fcns.h (susbt_octave_home): String length is size_t.

diff --git a/src/shared-fcns.h b/src/shared-fcns.h
--- a/src/shared-fcns.h
+++ b/src/shared-fcns.h
@@ -127,17 +127,17 @@ subst_octave_home (const std::string& s)
   std::string octave_home = get_octave_home ();
 
   std::string prefix = OCTAVE_PREFIX;
 
   retval = s;
 
   if (octave_home != prefix)
     {
-      octave_idx_type len = prefix.length ();
+      size_t len = prefix.length ();
 
       if (s.substr (0, len) == prefix)
         retval.replace (0, len, octave_home);
     }
 
   if (dir_sep_char != '/')
     std::replace (retval.begin (), retval.end (), '/', dir_sep_char);
 
